Sources/TuistKit/Linter/TargetLinter.swift

```diff
@@ -82,16 +82,12 @@ class TargetLinter: TargetLinting {
     }
 
     fileprivate func lintLibraryHasNoResources(target: Target) -> [LintingIssue] {
-        
-        if target.product == .dynamicLibrary || target.product == .staticLibrary {
-            return [ ]
+        if target.product != .dynamicLibrary && target.product != .staticLibrary {
```

Comment:
Should add `staticFramework` here until support is added for resources

Reply:
Yes. I think the resources support can be in a follow-up PR.